src/clr-addons/view-edit-section/view-edit-section.ts

```diff
@@ -99,4 +101,8 @@ export class ClrViewEditSection {
       this.toggleCollapsed(true);
     }
   }
+
+  submitButtonState(): ClrLoadingState {
```

Comment:
```suggestion
  getSubmitButtonLoadingState(): ClrLoadingState {
```

feel free to choose a different name, but at first glance it looked like the method submits something 😄